### PR TITLE
#19 init Internationalized and multilingual support [es, en]

### PR DESCRIPTION
**Pull request by alejandroaguilarhiguera**

### CODE REVIEW

File: components/ImageUploader.tsx [hunk 1]
```diff
@@ -200,10 +201,10 @@ export default function ImageUploader({ target, id, buttonMsg, handleAvatarChang
               </div>
               <div className="mt-5 sm:mt-4 sm:flex sm:flex-row-reverse">
                 <button type="button" className="btn btn-primary" onClick={showCroppedImage}>
-                  Save
+                  <FormattedMessage id="save" defaultMessage="Save" />
                 </button>
                 <button onClick={closeImageUploadModal} type="button" className="btn btn-white mr-2">
-                  Cancel
+                  <FormattedMessage id="cancel" defaultMessage="Cancel" />
```
Comment (PeerRich) [inline components/ImageUploader.tsx:207]: we should abstract this into a shorter component 

something i've seen people do is


```

<T>Hello</T>

```

Comment (alejandroaguilarhiguera) [reply components/ImageUploader.tsx]: this is fixed

Comment (PeerRich) [reply components/ImageUploader.tsx]: oh, not the way I was thinking off. you've just renamed the import. technically makes it shorter but I was thinking of a different implementation:

In a previous project that I worked on we had a wrapper function `<T>Hello</T>` that would create an id by lowercasing and removing spaces and special characters.

you write: `<T>Hello</T>` and it generates: `<FormattedMessage id="hello" defaultMessage="Hello" />`

the speed of writing strings is the most important part of i18n. an annoyingly long API will slow down development.

Comment (PeerRich) [reply components/ImageUploader.tsx]: also, you're still directly importing react-intl in every component vs. abstracting it into components/T.jsx

File: lib/toCamelCase.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Convert a CamelCase string to kebab-case
+export const toCamelCase = function toCamelCase(string = ""): string {
+  console.log({ string });
```
Comment (jordanhart) [inline lib/toCamelCase.ts:3]: We probably don’t want a console.log here

**Review by jordanhart — CHANGES_REQUESTED** [review]

File: lib/toCamelCase.ts (hunk 2 above)
Comment (alejandroaguilarhiguera) [reply lib/toCamelCase.ts]: fixed on commit 2db8f61b4f8dc8e492e9c541ae7bfb288cd17f76

**Pull request closed without merging**